### ISSUE TITLE
Scheduler does not stop after stopping

### ISSUE THREAD

**Issue by smgrol** [Shuffle/Shuffle#1406]
**Describe the bug**
Workflow is launched by a scheduler trigger.
If the scheduler is stopped, the workflow continues to run according to the old timer.  
**To Reproduce**
1. Create new workflow `test_scheduler` .
2. Add trigger `Scheduler` to workflow and set interval 15 sec.
3. Add app `Shuffle Tools` to workflow with action `Repeat back to me` and call text `Hello world!` .
4. Start scheduler and save workflow
5. ....wait a couple of minutes.....
6. Stop scheduler
7. Open page `Show executions` . 
8. Workflow still executiong by timer.
...

**Expected behavior**
After stopping the timer, the workflow stops working.

**Screenshots**

https://github.com/Shuffle/Shuffle/assets/70931518/5fdb1a3b-3e02-495c-bcef-16c9af2363cc

**Comment by smgrol** [comment]
Yes, if I stop new created workflow with trigger `schedule` from **admin?tab=schedules** , then trigger stopped and workflow no working. Thanks!


But now I have 2 problem workflows with a "schedule" trigger. These schedules were stopped from the workflow interface, but the workflows were running.

On this tab **admin?tab=schedules** both workflows have a button "Start Schedule" .
Both workflows have a scheduler disabled from web interface.
But both workflows processes are currently running.

It is worth noting th 2 workflows was imported from another installation of Shuffle.

**Comment by frikky** [comment]
> Yes, if I stop new created workflow with trigger `schedule` from **admin?tab=schedules** , then trigger stopped and workflow no working. Thanks!
> 
> But now I have 2 problem workflows with a "schedule" trigger. These schedules were stopped from the workflow interface, but the workflows were running.
> 
> On this tab **admin?tab=schedules** both workflows have a button "Start Schedule" . Both workflows have a scheduler disabled from web interface. But both workflows processes are currently running.
> 
> It is worth noting th 2 workflows was imported from another installation of Shuffle.

Heyo!

Would you be up for a call to look into this a bit further?

**Comment by melv33n** [comment]
Hi, this happening again to me after updating to 2.0.0 and rebuilding containers. Schedules are 'stop' on both interfaces but they keep running. Now it's happening to ALL workflows after updating.


> On this tab **admin?tab=schedules** both workflows have a button "Start Schedule" . Both workflows have a scheduler disabled from web interface. But both workflows processes are currently running.

I tried doing Start->Stop on one of them but shadow shcedule keeps running. 

@frikky , time ago i had a support session  where they tell me to export and import the workflow failing but now it's happening to ALL of them. I don't think doing that to +80 WFs is the solution. Instead we could try to debug.

**Comment by frikky** [comment]
> Hi, this happening again to me after updating to 2.0.0 and rebuilding containers. Schedules are 'stop' on both interfaces but they keep running. Now it's happening to ALL workflows after updating.
> 
> > On this tab **admin?tab=schedules** both workflows have a button "Start Schedule" . Both workflows have a scheduler disabled from web interface. But both workflows processes are currently running.
> 
> I tried doing Start->Stop on one of them but shadow shcedule keeps running.
> 
> [@frikky](https://github.com/frikky) , time ago i had a support session where they tell me to export and import the workflow failing but now it's happening to ALL of them. I don't think doing that to +80 WFs is the solution. Instead we could try to debug.

Wtf, weird we haven't heard about this elsewhere. Schedule a call asap so we can sort it out please: https://drift.me/frikky
